public/css/calendar.less

```diff
@@ -168,6 +178,50 @@
     border-left: none;
   }
 
+  .step {
+    grid-column-end: span @columnsPerDay;
+    grid-row-end: span @rowsPerHour;
+    position: relative;
+  }
+
+  .extra-count {
+    height: auto;
```

Comment:
That's the same in every mode, although not in the day mode, yeah. But there is no extra count anyway, so I don't think the rule hurts if applied to every grid. The position isn't that much of a deal as well, I guess.

Reply:
instead, I think, it would be better to set the height of  anchor `a` to 100% if it is not belonging to class `extra-count`.